### PR TITLE
update readme to use new theme

### PR DESCRIPTION
**Pull request by amanmdesai**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your pull request.

\- HSF Training
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,95 @@
...
+repository. For making your contribution, we use the GitHub flow, which is
+nicely explained in the chapter [Contributing to a Project][progit] in Pro Git
+by Scott Chacon.
+Look for the tag [![good_first_issue]][gfi-badge], which marks particularly simple issues to get you started.
 
-Current maintainers of this lesson are
+<!-- END CENTRALLY MAINTAINED SECTION -->
+## 💖 Authors
 
-* Meirin Oan Evans
```
Comment (klieret) [inline README.md:28]: we should keep mentioning Meirin until we properly add all the authors

**Review by klieret — CHANGES_REQUESTED** [review]
Thanks a lot!

File: README.md (hunk 1 above)
Comment (amanmdesai) [reply README.md:28]: @klieret  thanks for the comment. I have updated the readme.

**Comment by amanmdesai** [comment]
Hi @klieret this PR is ready for re-review
thanks

**Review by klieret — APPROVED** [review]
Thanks!

**Pull request merged**